archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/Archaius2BackportModule.java

```diff
@@ -0,0 +1,41 @@
+package com.netflix.archaius.bridge;
+
+import javax.inject.Singleton;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.Scopes;
+import com.netflix.archaius.ConfigProxyFactory;
+import com.netflix.archaius.api.Config;
+import com.netflix.archaius.api.config.SettableConfig;
+import com.netflix.config.ConfigurationManager;
+
+/**
+ * Install this module so that an injectable {@link ConfigProxyFactory} can be backed by the static
+ * Archaius1 {@link ConfigurationManager}.  Use this for legacy applications that use Archaius1 and
+ * haven't yet update to use Archaius2 as its configuration solution.
```

Comment:
Be good to add comments on how this interact (or not) with StaticArchaiusBridgeModule.